pkg/fpga/bitstream/bitstream.go

```diff
@@ -47,9 +45,9 @@ func GetFPGABitstream(bitstreamDir, region, afu string) (File, error) {
 // Open bitstream file, detecting type based on the filename extension.
 func Open(fname string) (File, error) {
 	switch filepath.Ext(fname) {
-	case ".gbs":
+	case fileExtensionGBS:
```

Comment:
IMHO code would be clearer if this `Open()` function would be inlined into `GetFPGABitstream()` loop.  That way one could more easily see that loop handles all cases that it ranges through.

Reply:
Looks reasonable to me but it could be handled in another PR.